migration/idempotent/007-tracing-tags.sql

```diff
@@ -209,20 +210,21 @@ the built-in jsonb. It is the same as its jsonb_ namesake, but relies on tag_map
 
 DO $do$
 BEGIN
-	CREATE OPERATOR ps_trace.<> (
+	DROP OPERATOR IF EXISTS ps_trace.<> (_ps_trace.tag_v, _ps_trace.tag_v)  CASCADE;
```

Comment:
Is there any reason why these `DROP` statements don't work in an incremental migration? I'm just concerned that we're going to accidentally get something wrong by removing these at some point in the future.

Reply:
That's a good question. I was thinking about this as well. Ultimately, I'd prefer related code to be in one place, so it's easier to make changes.
Unfortunately it's hard to achieve that in the current structure of the repo.

Reply:
I would argue that having the DROP statements in an idempotent script makes it _harder_ to make changes.

Reply:
I see your point. Let me do some adjustments